formstack-consents/src/main/scala/com/gu/identity/formstackconsents/Lambda.scala

```diff
@@ -0,0 +1,19 @@
+package com.gu.identity.formstackconsents
+
+import com.gu.identity.globalConfig.DevConfig
+
+object Lambda extends App {
+
+  val newsletters: List[Newsletter] = List(Holidays, Students, Universities, Teachers, Masterclasses, SocietyWeekly, EdinburghFestivalDataCollection)
+
+  def handler(): Unit = {
+    // TODO: look into integrating parameter store through cloudformation
```

Comment:
Config is currently set within the project but once I look into cloudformation and parameter store, I will change this.